testsuites/CBLTester/p2p_tests/test_p2p_scopes_collections.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from keywords import attachment
+from CBLClient.PeerToPeer import PeerToPeer
+from CBLClient.Collection import Collection
+from CBLClient.Replication import Replication
+from keywords.utils import random_string
+
+
+@pytest.mark.p2p
+@pytest.mark.listener
+@pytest.mark.parametrize("num_of_docs, continuous, replicator_type, attachments, endPointType, scope, collection", [
+    (100, False, "push_pull", False, "URLEndPoint", random_string(6), random_string(6))
+])
+def test_peer_to_peer(params_from_base_test_setup, server_setup, num_of_docs, continuous, replicator_type, attachments, endPointType, scope, collection):
+    """
+        @summary: peer1 -> Peer2
```

Comment:
Would you be able to also add these lines to the actual operations the tests is doing? It's just easier to understand what the test is doing like that